build.gradle.kts

```
@@ -246,25 +249,23 @@ oci {
                 workingDirectory = "/opt/hivemq"
             }
             layers {
-                layer("hivemq") {
+                layer("main") {
                     contents {
                         into("opt") {
-                            from("docker/docker-entrypoint.sh") { filePermissions = 0b111_101_101 }
-                            permissions("hivemq/", 0b111_111_000)
+                            from("src/oci/docker-entrypoint.sh") { filePermissions = 0b111_101_101 }
+                            permissions("hivemq/", 0b111_111_101)
                             into("hivemq") {
-                                filePermissions = 0b110_100_000
-                                directoryPermissions = 0b111_101_000
-                                permissions("**/*.sh", 0b111_101_000)
-                                permissions("conf/", 0b111_111_000)
-                                permissions("conf/config.xml", 0b110_110_000)
-                                permissions("conf/logback.xml", 0b110_110_000)
-                                permissions("data/", 0b111_111_000)
-                                permissions("extensions/", 0b111_111_000)
-                                permissions("extensions/*/", 0b111_111_000)
-                                permissions("extensions/*/hivemq-extension.xml", 0b110_110_000)
-                                permissions("log/", 0b111_111_000)
+                                permissions("**/*.sh", 0b111_101_101)
+                                permissions("conf/", 0b111_111_101)
+                                permissions("conf/config.xml", 0b110_110_100)
+                                permissions("conf/logback.xml", 0b110_110_100)
+                                permissions("data/", 0b111_111_101)
+                                permissions("extensions/", 0b111_111_101)
+                                permissions("extensions/*/", 0b111_111_101)
```

Might make sense to add the TODO here as well
```suggestion
                                permissions("extensions/*/", 0b111_111_101) // TODO remove allow-all-extension?
```